hack/templates/excluder-template

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
contrib/yum seems most clear indication of what the files are used for but given this applies to both yum and dnf maybe not the best choice? contrib/excluder seems overly generic to me but a good candidate for avoiding that problem.

Just make sure to add a README.md that describes the use of the files in the contrib dir you choose.